url-shortener/android/build.gradle

```
@@ -1,15 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
-
 buildscript {
-    ext.kotlin_version = '1.3.61'
+    ext.kotlin_version = "1.5.10"
     repositories {
         google()
         jcenter()
```

WDYT aboud remove `jcenter()` from here and from `repositories` below.
It will be shutdown soon. Replace it by `mavenCentral`
```suggestion
        mavenCentral()
```